mibidata/panels.py

```diff
@@ -56,10 +56,20 @@ def read_csv(path):
         # Combine and convert 'Mass' column to int
         combined = pd.concat(df, ignore_index=True)
         combined['Mass'] = combined['Mass'].astype(np.int64)
-        return _merge_masses(combined)
+        return merge_masses(combined)
 
 
-def _merge_masses(df):
+def merge_masses(df):
+    """Merges 'Target' cells of a dataframe with the same 'Mass' value.
+
+    Args:
+        dataframe: A dataframe of the CSV file containing columns 'Mass' and
```

Comment:
The first part before the ':' should match the name of the parameter in the function definition, i.e. `df` in this case.